rbac/api/cross_access/inventory_api_dual_write_cross_access_handler.py

```
@@ -0,0 +1,327 @@
+                    info={
+                        "user_id": str(self.cross_account_request.user_id),
+                        "roles": [role.uuid for role in self.cross_account_request.roles.all()],
+                        "target_org": self.cross_account_request.target_org,
+                        "org_id": self.cross_account_request.target_org,
+                    },
+                    partition_key=PartitionKey.byEnvironment(),
+                    remove=self.relations_to_remove,
+                    add=deduplicated_add,
+                ),
+            )
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)